src/lang/res_pt-BR.rc

```diff
@@ -0,0 +1,644 @@
+/********************************************************************
+
+   res_pt-BR.rc
+
+   Copyright (c) Microsoft Corporation. All rights reserved.
+   Licensed under the MIT License.
+
+********************************************************************/
+
+LANGUAGE LANG_PORTUGUESE, SUBLANG_PORTUGUESE_BRAZILIAN
+
+FRAMEMENU MENU
+BEGIN
+    POPUP       "&Arquivo"
```

Comment:
This is the same accelerator as used for `Ajuda`, making opening it with the keyboard a pain.  The original used `?` instead of `Ajuda` presumably for this reason.  Is there a common convention for representing these two, since both seem very common?